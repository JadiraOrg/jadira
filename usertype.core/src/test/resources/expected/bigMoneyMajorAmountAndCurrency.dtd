<!ELEMENT dataset (
    BIGMONEYMAJORAMOUNTANDCURRENCY*)>

<!ELEMENT BIGMONEYMAJORAMOUNTANDCURRENCY EMPTY>
<!ATTLIST BIGMONEYMAJORAMOUNTANDCURRENCY
    ID CDATA #REQUIRED
    MY_CURRENCY CDATA #IMPLIED
    MY_AMOUNT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

