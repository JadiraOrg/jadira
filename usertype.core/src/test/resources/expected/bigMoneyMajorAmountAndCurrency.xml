<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyMajorAmountAndCurrency.dtd">
<dataset>
  <bigMoneyMajorAmountAndCurrency ID="0" MY_CURRENCY="USD" MY_AMOUNT="100" NAME="test_0"/>
  <bigMoneyMajorAmountAndCurrency ID="1" MY_CURRENCY="USD" MY_AMOUNT="100" NAME="test_1"/>
  <bigMoneyMajorAmountAndCurrency ID="2" MY_CURRENCY="EUR" MY_AMOUNT="0" NAME="test_2"/>
  <bigMoneyMajorAmountAndCurrency ID="3" MY_CURRENCY="EUR" MY_AMOUNT="0" NAME="test_3"/>
  <bigMoneyMajorAmountAndCurrency ID="4" NAME="test_4"/>
</dataset>